lib/sequenceserver/blast/job.rb

```diff
@@ -24,13 +24,23 @@ def initialize(params)
             @databases = Database[params[:databases]]
             @advanced  = params[:advanced].to_s.strip
             @options   = @advanced + defaults
+            @num_threads = config[:num_threads]
+            @query_length = query_size
+            @total_database_length = ncharacters_total
```

Comment:
Can we add a comment above this code block indicating these are for analytics and not per se required to run a job.

Reply:
Sure, I think that's very helpful.